### DISCUSSION TITLE
View factor

### DISCUSSION THREAD

**Discussion by Sadeghi6872** [firemodels/fds#13472] [General]
Hi All, 

Does FDS consider the view factor in its default radiation model?

thanks.

**Comment by mcgratta** [comment]
A view factor calculation is a simplified way of doing radiation transport. FDS does radiation transport by solving the radiation transport equation for approximately 100 discrete directions. The end result of an FDS simulation should give you the same result as a view factor calculation, but it does not explicitly calculate view factors.
